next/pages/guidelines/color/overview.tsx

```diff
@@ -0,0 +1,200 @@
+                    return (
+                        <tr key={key}>
+                            <td className="v-mid pt2 pr4 tl s_nowrap">
+                                <span className="pv1 flex items-center br2 b">
+                                    {list[key].title}
+                                </span>
+                            </td>
+                            <td className="v-mid pt2 s_nowrap black-300">
+                                {list[key].description}
+                            </td>
+                        </tr>
+                    );
+                })}
+            </tbody>
+        </table>
+    );
+}
+
+export default function Overview({ layoutProps }: InferGetStaticPropsType<typeof getStaticProps>) {
+    return (
+        <ContentPage
+            title="Overview"
+            description="A simple system for interfaces and illustrations"
+            layoutProps={layoutProps}
+        >
+            <P>
+                Colors play a crucial role in communicating status, guidance, and providing visual
+                cues. Color should be used sparingly to drive focus to moments that matter. Color
+                should not be used to add personality or flair.
+            </P>
+
+            <div>
+                <H2>Color themes</H2>
+                <P>
+                    Our use of color can be defined in two major types classified as brand and
+                    feedback colors.
+                </P>
+            </div>
+
+            <div>
+                <H3>Brand colors</H3>
+                <P>
+                    These colors are used to create a consistent visual identity and evoke specific
+                    emotions or associations within a design system.
+                </P>
+                {colorThemeTable({ type: 'brand' })}
+            </div>
+
+            <div className="mb5">
+                <H3>Feedback colors</H3>
+                <P>
+                    These colors are used to help re-enforce important moments in the user journey
+                    that suggest or require additional guidance, gather user input, and improve the
+                    user experience.
+                </P>
+                {colorThemeTable({ type: 'feedback' })}
+            </div>
+
+            <Link href="/guidelines/color/palette/">See all colors</Link>
+
+            <div className="mb5">
+                <H2>Usage</H2>
+                <P>
+                    Color usage is arranged into four high-level categories background, border, text
+                    and icon. Color usage patterns are represented by their intended use case.
+                </P>
+                <ExampleBox>
+                    <div className="tc">
+                        <img src={images.usage.src.src} width="375px" alt={images.usage.alt} />
+                    </div>
+                </ExampleBox>
+                {overviewTable({ list: usageContent })}
```

Comment:
This works, but the more traditional way to do it with React is to do rename `overviewTable` to `OverviewTable` and do: `<OverviewTable list={usageContent} />`.